src/lib.rs

```diff
@@ -6,4 +6,4 @@ pub use flatdata::{Archive, FileResourceStorage};
 
 mod osmflat;
 
-pub use osmflat::*;
+pub use crate::osmflat::*;
```

Comment:
Btw. I would prefer not to use *. In Python this is a total antipattern. And in the end we are not paying for the number of lines.

Reply:
Here we are exporting everything in the generated file... seems to make a lot of sense?